pegasus/sites.v3/code.org/public/hourofcode2022.haml

```diff
@@ -0,0 +1,103 @@
+---
+title: Explore, Play, and Create with Computer Science
+theme: responsive
+---
+
+%link{href: "/css/generated/hoc2022-landing-page.css", rel: "stylesheet"}
+
+-# Import FontAwesome Pro
+%script{src: "https://kit.fontawesome.com/ea2d9d4413.js", crossorigin: "anonymous"}
+
+-# FontAwesome icons
+- icon_rocket = "fa-regular fa-rocket"
+- icon_futbol = "fa-regular fa-futbol"
+- icon_paint = "fa-regular fa-paintbrush-pencil"
+
+%header
+  .wrapper
+    %span=I18n.t(:og_title_here)
+    %h1=I18n.t(:hoc2022_banner_title)
+    %h2=I18n.t(:try_it_yourself)
+    %nav
+      %ul
+        %li
+          %i{class: "#{icon_rocket}"}
+          %a{href: "#explore"}
+            =I18n.t(:hoc2022_landing_page_explore_title)
+        %li
+          %i{class: "#{icon_futbol}"}
+          %a{href: "#play"}
+            =I18n.t(:hoc2022_landing_page_play_title)
+        %li
+          %i{class: "#{icon_paint}"}
+          %a{href: "#create"}
+            =I18n.t(:hoc2022_landing_page_create_title)
+  %img.left{src: "/images/hoc2022-landing-page-banner-left.svg", alt: I18n.t(:hoc2022_landing_page_banner_alt_left)}
+  %img.right{src: "/images/hoc2022-landing-page-banner-right.svg", alt: I18n.t(:hoc2022_landing_page_banner_alt_right)}
+
+%main
+  -# Hiding the NASA banner until post-launch due to mktg reasons
+  -# %aside
+    %img{src: "/images/nasa-logo.svg", alt: "NASA logo"}
+    %h1
+      Copy from NASA goes here
+    %a{href: "#", target: "_blank", rel: "noopener noreferrer"}
+      %button
+        Learn more
+
+  %a{id: "explore"}
+  %section.explore
+    .wrapper
+      .top-wrapper
+        %div
+          %i{class: "#{icon_rocket}"}
+          %h1=I18n.t(:hoc2022_landing_page_explore_title)
+          %p=I18n.t(:hoc2022_landing_page_explore_description)
+        %figure
+          %div
+            %iframe{src: "https://www.youtube.com/embed/gDpNXnrFvFE", allowFullScreen: 'true', frameborder: '0'}
```

Comment:
@megcrenshaw I just updated this w/ the new NASA video! (Probs doesn't need extra review, but I wanted to show you since it has another very cool astronaut in it 😎)

Reply:
Oh hurray! Thank you for the callout!! 🥳